nms/config/config.ts

```diff
@@ -12,13 +12,20 @@
  */
 
 import fs from 'fs';
+import {Options as RateLimitOptions} from 'express-rate-limit';
 import {getValidLogLevel} from '../shared/logging';
 
 export const DEV_MODE = process.env.NODE_ENV !== 'production';
 export const LOG_FORMAT = DEV_MODE ? 'shell' : 'json';
 export const LOG_LEVEL = getValidLogLevel(process.env.LOG_LEVEL);
 export const LOGGER_HOST = process.env.LOGGER_HOST || 'fluentd:9880';
 export const API_HOST = process.env.API_HOST || 'magma_test.local';
+export const RATE_LIMIT_CONFIG: Partial<RateLimitOptions> = {
+  windowMs: Number(process.env.RATE_LIMIT_WINDOW) || 15 * 60 * 1000, // 15 min default
+  limit: Number(process.env.RATE_LIMIT_COUNT) || 100,
+  standardHeaders: 'draft-7',
+  legacyHeaders: false,
```

Comment:
A default message is expected to be included.

Reply:
The default is "Too many requests, please try again later.", what do you recommend?

Reply:
Please go ahead and add the above message provided by you to the default.